File: samples/golang/1_sync_code_with_counter.go
package main

import "fmt"

var counter = 0

func main() {
	incrementCounter()

	fmt.Printf("The final value of counter is: %d\n", counter)
}

func incrementCounter() {
	for i := 0; i < 1000000; i++ {
		counter++
	}
}

File: samples/csharp/1_sync_code_with_counter.cs
using System;

public class Program
{
    static int counter = 0;

    static void Main(string[] args)
    {
        IncrementCounter();

        Console.WriteLine($"The final value os counter is: {counter}");
    }

    static void IncrementCounter()
    {
        for (int i = 0; i < 1000000; i++)
	        counter++;
    }
}
